doc/MiniLanguageTutorial.md

```diff
@@ -120,17 +126,21 @@ Mini has the following types:
 
 > either the contained type or "None<*type*>", must be unwrapped to compare with inner type
 
-[`impure`] `func` ( *type1, type2, ...*) [-> *returntype*]
+[ `view` | `write` | `public` ]* `func` ( *type1, type2, ...*) (-> *returntype*)?
 
 > a reference to a function
 
+[ `view` | `write` ] `closure` `(` *type*, *type*, ... `)` (`->` *returntype*`)?
+
+> a closure type, acts like a function except that it can capture data from its contained scope.
```

Comment:
There's no such thing as the "closure" type. Funcs and closures are the same in mini

Reply:
While there is no distinct type, there is a type alias available in the parser. See:  https://github.com/OffchainLabs/arb-os/blob/documentation/src/mini.lalrpop#L341

Reply:
I'll reword this to include the fact that it is an alias.